lib/AST/ExprClassification.cpp

```diff
@@ -445,6 +445,8 @@ static Cl::Kinds ClassifyInternal(ASTContext &Ctx, const Expr *E) {
   case Expr::NullaryBoundsExprClass:
   case Expr::RangeBoundsExprClass:
     llvm_unreachable("should not classify bounds expressions");
+  case Expr::PackExprClass:
+    return Cl::CL_PRValue;
```

Comment:
It would be good to add a comment about this.

Reply:
Done